server/scores.js

```diff
@@ -15,16 +15,56 @@ const connectionPromise = MongoClient
   .connect(mongoUrl, { promiseLibrary: Promise, useNewUrlParser: true })
   .then(client => client.db().collection('scores'))
 
+function _validateScore (score) {
+  let retError = ['ok', '']
+  if (typeof score.teamNumber !== 'number') {
+    retError[0] = 'loud-fail'

```

Comment:
To can simply return it in each if statement, instead of staying it through all the ifs after one had validated